src/backend/core/api/viewsets.py

```diff
@@ -316,6 +317,15 @@ class TeamAccessViewSet(
     queryset = models.TeamAccess.objects.all().select_related("user")
     list_serializer_class = serializers.TeamAccessReadOnlySerializer
     detail_serializer_class = serializers.TeamAccessSerializer
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter)
+    ordering_fields = [
+        "role",
+        "user__email",
+        "user__name",
+        "user__main_identity__email",
+        "user__main_identity__name",
```

Comment:
Syntax is correct but it doesn't seem to be working because it should refer a field.